tests/Application/Matching/AdapterTest.php

```diff
@@ -111,39 +115,43 @@ public function testItBailsOutAndReleasesFundsIfRetryingDoesntWorkDueToConcurren
         // let's assume another thread is causing the funds to reduce by 30 pounds just
         // after each time we increase it by 30 pounds.
```

Comment:
```suggestion
        // let's assume another thread is causing the funds to reduce by 40 pounds just
        // after each time we increase it by 30 pounds – via `subtractAmountWithoutSavingToDB()`
        // recovery calling `incrBy()` with overspent amount.
```

I think half this comment is talking about the amended value and the second half is about the subtracted £30 getting put back. But I'm not I fully understood it – so also proposing extra detail to confirm whether I have.

Reply:
I think your understanding is right, this is a good change to the comment. It was 30 but I increased it to 40 to see how that effected the result.